src/pane-container-element.coffee

```diff
@@ -36,6 +36,27 @@ class PaneContainerElement extends HTMLElement
   focusPaneViewOnRight: ->
     @nearestPaneInDirection('right')?.focus()
 
+  moveActiveItemToPaneAbove: (params) ->
+    @moveActiveItemToNearestPaneInDirection('above', params)
+
+  moveActiveItemToPaneBelow: (params) ->
+    @moveActiveItemToNearestPaneInDirection('below', params)
+
+  moveActiveItemToPaneOnLeft: (params) ->
+    @moveActiveItemToNearestPaneInDirection('left', params)
+
+  moveActiveItemToPaneOnRight: (params) ->
+    @moveActiveItemToNearestPaneInDirection('right', params)
+
+  moveActiveItemToNearestPaneInDirection: (direction, params) ->
+    destPane = @nearestPaneInDirection(direction)?.getModel()
+    return unless destPane?
+    if params?.keepOriginal
+      @model.copyActiveItemToPane(destPane)
+    else
+      @model.moveActiveItemToPane(destPane)
+    destPane.focus()
```

Comment:
Should we call `destPane.focus()` on both `copyActiveItemToPane` and `moveActiveItemToPane` (in `PaneContainer`) instead?
Once I get some feedback on this, I'll add some specs for those methods.